Parsers/ASimProcessEvent/test/SentinelOne_vimProcessCreate_DataTest.csv

```diff
@@ -0,0 +1,23 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 5185 records (100.0%) for field [EventProduct] of type [Enumerated]: [""SentinelOne""] (Schema:ProcessEvent)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 5185 records (100.0%) for field [EventVendor] of type [Enumerated]: [""SentinelOne""] (Schema:ProcessEvent)"
+"(1) Warning: Empty value in 2 records (0.04%)  in mandatory field [TargetProcessName] (Schema:ProcessEvent)"
+"(1) Warning: Empty value in 4849 records (93.52%)  in mandatory field [ActorUsername] (Schema:ProcessEvent)"
```

Comment:
why sample data doesn't have any ActorUsername? its one of the mandatory field

Reply:
Updated sample data